integration/node/manager_test.go

```diff
@@ -355,12 +372,12 @@ func (s *ManagerTestSuite) TestRaceConditions() {
 			s.T().Logf("PopulateStaticPeers(), error: %v", s.NodeManager.PopulateStaticPeers())
 			progress <- struct{}{}
 		},
-		func(config *params.NodeConfig) {
-			log.Info("ResetChainData()")
-			_, err := s.NodeManager.ResetChainData()
-			s.T().Logf("ResetChainData(), error: %v", err)
-			progress <- struct{}{}
-		},
+		// func(config *params.NodeConfig) {
```

Comment:
The same comment about commenting a code (see above)

Reply:
The same answer :) `ResetChainData()` removes all data and I'd like to avoid it. We should collect all `ResetChainData()`  tests in one place and initialize the node with a different data dir.